test/exe/terminate_handler_test.cc

```diff
@@ -0,0 +1,13 @@
+#include "exe/terminate_handler.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+
+TEST(TerminateHandler, HandlerInstalledTest) {
+  TerminateHandler handler;
+  EXPECT_DEATH([]() -> void { std::terminate(); }(),
```

Comment:
note: please merge master, you need to use a different macro now for death tests (search code, don't remember off the top of my head)

Reply:
I think you want EXPECT_DEATH_LOG_TO_STDERR